### ISSUE TITLE
Client not recognizing command

### ISSUE THREAD

**Issue by Swilloughby-github** [rsanchez-wsu/jfiles#210]
So I try to input a FILE command while testing out the program. The program will not recognize the command, and instead just endlessly outputs the "Command not found" string until the program is ended. This has to be due to a recent commit. Obviously, this issue can be considered fixed when the program can at least detect one of the integrated commands and execute accordingly.

**Comment by rsanchez-wsu** [comment]
@Swilloughby-github I haven't looked to see what has been merged into your branch, but this commit may be related: c24929e

**Comment by LoganRickert** [comment]
I think this issue has been resolved. The issue is not present on master and it seems the way commands are handled now have been changed at some point, though I was unable to track down when.

I tested out the server with the following both commented out and not commented out:

```
case "FILE":
    break;
```

in server.JFilesServer and received the following output:

```
>file x
>
```

The file command and invalid commands seem to no longer break the server.

**Comment by rsanchez-wsu** [comment]
@LoganRickert This is likely a problem on the one of the team1 branches.  Most likely it is team1-dev.  I would check there.

**Comment by LoganRickert** [comment]
I had a look at all of the team1's branches and it seems their code is pretty different from master. They don't even support the same commands.

**Comment by rsanchez-wsu** [comment]
Yes, their code is quite different.  Team 1 was the one that had the student in California who re-implemented the entire networking component.  I intended to review the code and try to integrate over Christmas break, but that didn't happen.  Feel free to take a crack at it or at least to look at the code in depth and make some recommendations.

**Comment by rsanchez-wsu** [comment]
@LoganRickert if you are interested in working on this, I suggest you get in touch with @joyceje here or in Slack (he has joined our team and joyceje is his username in Slack as well). He was the lead for team 1 and he has the best understanding of the code on those branches.
